packages/backend-core/src/Endpoint/EndpointGroup.ts

```diff
@@ -0,0 +1,104 @@
+import Endpoint, { CtxFn, Method } from "./Endpoint"
+import Router from "@koa/router"
+
+type ArrayOneOrMore<T> = {
```

Comment:
That's a funny way to define this. I've usually seen it like `type ArrayOneOrMore<T> = [T,...T[]] `

Reply:
That is actually a lot nicer lol